### DISCUSSION TITLE
Why can't I make the window blur with the blur-my-shell gnome extension?

### DISCUSSION THREAD

**Discussion by rafli-sss** [wezterm/wezterm#4494] [Q&A] [answered]
### What Operating System(s) are you running on?

Linux Wayland

### Which Wayland compositor or X11 Window manager(s) are you using?

Mutter

### WezTerm version

20231023_015614_84ad0400

### Ask your question!

Extension : [blur-my-shell](https://extensions.gnome.org/extension/3193/blur-my-shell/)
I've set the opacity below 1 but it still doesn't blur.
![Screenshot from 2023-10-27 10-22-40](https://github.com/wez/wezterm/assets/92916692/c1df86ac-2169-498a-8f8e-03cab4516303)

A list of window to blur :
![Screenshot from 2023-10-27 09-53-24](https://github.com/wez/wezterm/assets/92916692/40814b25-fc5f-4564-b5b0-4a96af864094)

**Comment by rafli-sss** [comment, marked as answer]
Solved, change to :
org.wezfurlong.wezterm

**Comment by hoanhxlyn** [comment]
For me i also have to disable the `Enable all by default` option at `Applications` tab
